### PR TITLE
Add intervals to test

### PR DESCRIPTION
**Pull request by joecorall**
**GitHub Issue**: (link)

* Other Relevant Links (Google Groups discussion, related pull requests,
 Release pull requests, etc.)

# What does this Pull Request do?

A brief description of what the intended result of the PR will be and/or what
 problem it solves.

# What's new?
A in-depth description of the changes made by this PR. Technical details and
 possible side effects.

* Changes x feature to such that y
* Added x
* Removed y
* Does this change require documentation to be updated? 
* Does this change add any new dependencies? 
* Does this change require any other modifications to be made to the repository
 (i.e. Regeneration activity, etc.)? 
* Could this change impact execution of existing code?

# How should this be tested?

A description of what steps someone could take to:
* Reproduce the problem you are fixing (if applicable)
* Test that the Pull Request does what is intended.
* Please be as detailed as possible.
* Good testing instructions help get your PR completed faster.

# Additional Notes:
Any additional information that you think would be helpful when reviewing this
 PR.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/committers

### CODE REVIEW

File: tests/src/Kernel/EdtfUtilsTest.php [hunk 1]
```diff
@@ -49,6 +49,7 @@ class EdtfUtilsTest extends KernelTestBase {
     '1900-01-02T01:22:33Z' => [],
     '1900-01-02T01:22:33+' => ['The date/time \'1900-01-02T01:22:33+\' is invalid.'],
     '1900-01-02T01:22:33+05:00' => [],
+    '1900/1999' => [],
```
Comment (adam-vessey) [inline tests/src/Kernel/EdtfUtilsTest.php:52]: Is handled below: https://github.com/Islandora/controlled_access_terms/pull/118/files#diff-db5fc9b1d0aac630139ad5c4a17033653350c562c922c9b85624f8e97f5e0e2cR93 ... possibly?

Comment (joecorall) [reply tests/src/Kernel/EdtfUtilsTest.php:52]: Thanks - I [was just testing out something](https://islandora.slack.com/archives/CM5PPAV28/p1715620800498579) and this was easier than running the tests locally.

Comment (adam-vessey) [reply tests/src/Kernel/EdtfUtilsTest.php:52]: ... though, yeah looking bit more, guess it makes sense to test here for the validation, as it's more the extraction of an ISO 8601 value being tested below.

**Comment by joecorall** [comment]
Already handled - there's a form setting to allow/disallow intervals in a given field

**Pull request closed without merging**